packages/paste-website/src/pages/core/index.mdx

```diff
@@ -0,0 +1,214 @@
+---
+title: Core
+description: The Core package includes all components and libraries in Paste.
+slug: /core/
+---
+
+import {graphql} from 'gatsby';
+import {Separator} from '@twilio-paste/separator';
+import {Text} from '@twilio-paste/text';
+import {Paragraph} from '@twilio-paste/paragraph';
+import {Box} from '@twilio-paste/box';
+import {Table, THead, TBody, Td, Th, Tr} from '@twilio-paste/table';
+import PackageJSON from '@twilio-paste/core/package.json';
+import {Disclosure, DisclosureContent, DisclosureHeading} from '@twilio-paste/disclosure';
+import {version as coreVersion, dependencies as coreDependencies} from '@twilio-paste/core/package.json';
+import Changelog from '@twilio-paste/core/CHANGELOG.md';
+import {DoDont, Do, Dont} from '../../components/DoDont';
+import {SidebarCategoryRoutes} from '../../constants';
+
+export const pageQuery = graphql`
+  {
+    mdx(fields: {slug: {eq: "/core/"}}) {
+      fileAbsolutePath
+      frontmatter {
+        slug
+        title
+      }
+      headings {
+        depth
+        value
+      }
+    }
+  }
+`;
+
+<ComponentHeader
+  isCore
+  name={props.pageContext.frontmatter.title}
+  categoryRoute={SidebarCategoryRoutes.CORE}
+  githubUrl="https://github.com/twilio-labs/paste/tree/main/packages/paste-core"
+  storybookUrl="?path=/story/"
+  packageStatus={[
+    {
+      node: {
+        data: {
+          Documentation: true,
+          Figma: 'not applicable',
+          Design_committee_review: 'yes',
+          Engineer_committee_review: 'yes',
+          Code: 'yes',
+          status: 'production',
+        },
+      },
+    },
+  ]}
+  data={[
+    {
+      node: {
+        name: PackageJSON.name,
+        description: PackageJSON.description,
+        version: PackageJSON.version,
+      },
+    },
+  ]}
+/>
+
+---
+
+ <contentwrapper>
+<content>
```

Comment:
Can you add the heading navigation in? This is a long page and I think it would be really helpful.

Reply:
Yesss thank you for catching this! Added it to the upgrade guide as well.